autoload/codefmt/mixformat.vim

```diff
@@ -0,0 +1,108 @@
+" Copyright 2022 Google Inc. All rights reserved.
+"
+" Licensed under the Apache License, Version 2.0 (the "License");
+" you may not use this file except in compliance with the License.
+" You may obtain a copy of the License at
+"
+"     http://www.apache.org/licenses/LICENSE-2.0
+"
+" Unless required by applicable law or agreed to in writing, software
+" distributed under the License is distributed on an "AS IS" BASIS,
+" WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+" See the License for the specific language governing permissions and
+" limitations under the License.
+
+let s:plugin = maktaba#plugin#Get('codefmt')
+let s:cmdAvailable = {}
+
+""
+" @private
+" Formatter: mixformat
+function! codefmt#mixformat#GetFormatter() abort
+  let l:formatter = {
+      \ 'name': 'mixformat',
+      \ 'setup_instructions': 'mix is usually installed with Elixir' .
+          \ '(https://elixir-lang.org/install.html). ' .
```

Comment:
Did you mean for this to be "with Elixir (https://...)" here with a space after "Elixir", or was "with Elixir(https://...)" intentional?

Reply:
Good catch.